csharp/Facebook.CSSLayout/CSSLayoutContext.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Facebook.CSSLayout
+{
+    public class CSSLayoutContext
```

Comment:
@emilsjolander is the intent then not to implement the regular `CSSNode` class and only implement the native class?